examples/data/shaders/glsl100/animation/forward.vs

```diff
@@ -0,0 +1,30 @@
+#version 100
```

Comment:
IIRC, we're targeting GLSL 330.

Reply:
We can't for the browser - we have to target GLSL 100 for webgl